From ea673b70908b301fc9901d804f141974491dcb3d Mon Sep 17 00:00:00 2001
From: Nitin Garg <nitin.garg@freescale.com>
Date: Wed, 25 Feb 2015 12:56:19 -0600
Subject: [PATCH] MLK-10259: Remove ARCH_MULTI_V6 config from v7 defconfig

Sync imx_v7_defconfig with imx_3.10. ARCH_MULTI_V6 is
not required.

Signed-off-by: Nitin Garg <nitin.garg@freescale.com>
---
 arch/arm/configs/imx_v7_defconfig | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index 910b2c0..27d7545 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -16,11 +16,8 @@ CONFIG_MODULE_UNLOAD=y
 CONFIG_MODVERSIONS=y
 CONFIG_MODULE_SRCVERSION_ALL=y
 # CONFIG_BLK_DEV_BSG is not set
-CONFIG_ARCH_MULTI_V6=y
 CONFIG_GPIO_PCA953X=y
 CONFIG_ARCH_MXC=y
-# CONFIG_MACH_MX31ADS is not set
-# CONFIG_MACH_BUG is not set
 CONFIG_MACH_IMX51_DT=y
 CONFIG_MACH_EUKREA_CPUIMX51SD=y
 CONFIG_SOC_IMX50=y
@@ -189,8 +186,8 @@ CONFIG_SPI_IMX=y
 CONFIG_GPIO_SYSFS=y
 CONFIG_POWER_SUPPLY=y
 CONFIG_SABRESD_MAX8903=y
-CONFIG_SENSORS_MAX17135=y
 CONFIG_IMX6_USB_CHARGER=y
+CONFIG_SENSORS_MAX17135=y
 CONFIG_SENSORS_MAG3110=y
 CONFIG_THERMAL=y
 CONFIG_CPU_THERMAL=y
-- 
1.8.0

